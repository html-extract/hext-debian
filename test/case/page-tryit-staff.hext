# match <div> having class "author"
<div class="author">
  # match <a> having attr. href, store it in "author_href"
  <a href:author_href>
    # match <img> having attr. src, store it in "author_img"
    <img src:author_img />
    # match <span> having class "name", store inner text
    # into "author_name"
    <span class="name" @text:author_name />
  </a>
</div>

