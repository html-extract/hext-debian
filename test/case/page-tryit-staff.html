<h1>Staff</h1>
<div class="staff-listing">
  <div class="author">
    <a href="/authors/bob.html">
      <img src="/images/authors/bob.jpg"
           alt="Bob" />
      <span class="name">Bob</span>
    </a>
  </div>
</div>
<div class="staff-listing">
  <div class="author">
    <a href="/authors/alice.html">
      <img src="/images/authors/alice.jpg"
           alt="Alice" />
      <span class="name">Alice</span>
    </a>
  </div>
</div>
<div class="staff-listing">
  <div class="author">
    <a href="/authors/carol.html">
      <img src="/images/authors/carol.jpg"
           alt="Carol" />
      <span class="name">Carol</span>
    </a>
  </div>
</div>

