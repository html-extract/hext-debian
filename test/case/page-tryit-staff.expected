{"author_href":"/authors/bob.html","author_img":"/images/authors/bob.jpg","author_name":"Bob"}
{"author_href":"/authors/alice.html","author_img":"/images/authors/alice.jpg","author_name":"Alice"}
{"author_href":"/authors/carol.html","author_img":"/images/authors/carol.jpg","author_name":"Carol"}
